features/step_definitions/salt_steps.rb

```diff
@@ -1,9 +1,36 @@
 # Copyright 2015 SUSE LLC
+$testsuite_hostname = `hostname -f`.chomp
 
 When(/^I get a content of a file "(.*?)"$/) do |filename|
   $output = sshcmd("cat #{filename}")
 end
 
+When(/^I issue local command "(.*?)"$/) do |command|
+  $local_output = `#{command}`
+  $output = {:stdout => $local_output}
+end
+
+When(/^I delete key of the testsuite hostname$/) do
```

Comment:
If you are referring to the client here, then use either what we use on other scenarios (I think we use on "this client") or in this case you can use "this minion".